### ISSUE TITLE
[1.15] Block loot table for shears should use tag

### ISSUE THREAD

**Issue by yanny7** [MinecraftForge/MinecraftForge#6514]
It will be very helpful if loot tables for shearing will be changed:

- now there is predicate for shear item
```
{
  "condition": "minecraft:match_tool",
  "predicate": {
    "item": "minecraft:shears"
  }
}
```
- with tag e.g. forge:shears developers can easely implement shearing for custom shears without adding new loot tables.
```
{
  "condition": "minecraft:match_tool",
  "predicate": {
    "tag": "forge:shears"
  }
}
```

**Comment by bloche1871** [comment]
There is another issue related to this that will not be solved by the above: items with the shears tag will trigger tripwires when breaking them, unlike vanilla shears.

**Comment by Shadows-of-Fire** [comment]
This issue was resolved by #6765
A separate issue should be opened relating to the tripwire problem, it is out of the scope of this issue.
Do note that the tripwire problem has been solved in 1.17 by https://github.com/MinecraftForge/MinecraftForge/pull/7997
